installer/pkg/common/certificates.go

```diff
@@ -0,0 +1,123 @@
+package common
```

Comment:
Please don't do this. Delegate this to cert-manager. One of the downsides of this is certificate rotation.

Reply:
Had conversation with @csweichel about this this morning, but we'd not considered cert rotation

Will amend. Thanks

Reply:
Actually, will rotation be needed? This is to replace the `genCa` and `genSignedCert` Helm functions. The values we have in there are hard-coded to a 365 day validity period and these aren't currently rotated. See https://github.com/gitpod-io/gitpod/blob/main/chart/templates/ws-manager-tlssecret.yaml#L51

For public certs, Cert Manager will do the job.

Reply:
@MrSimonEmms please check https://github.com/aledbf/gitpod-chart-cleanup/blob/main/charts/gitpod/templates/ws-manager-tlssecret.yaml
That is what is being used in the eks and gke guides.

The PRs to upstream the changes are a WIP https://github.com/gitpod-io/gitpod/pull/4592 and https://github.com/gitpod-io/gitpod/pull/4901